teku/teku-config.yaml

```diff
@@ -0,0 +1,7 @@
+network: auto
```

Comment:
i think we should use ETH2_NETWORK env variable here also, to be specific.

Reply:
Yeah I was conflicted about pointing everything at the one variable. The teku docs say everything except `mainnet` (default) and `auto` are deprecated. https://docs.teku.consensys.net/en/latest/Reference/CLI/Subcommands/Validator-Client/#network